### DISCUSSION TITLE
Using Annotations with the markdown list

### DISCUSSION THREAD

**Discussion by uPagge** [squidfunk/mkdocs-material#5103] [Q&A] [answered]
I tried to use the annotation inside the sheet, but I didn't figure out how to do it right. It turns out like this

```
- test (1)
- test 2
{ .annotate }

1. Annotation text
```
```
- test (1)
{ .annotate }
- test 2

1. Annotation text
```

None of the options works. Can anyone tell me what can be done or is it not yet supported?

**Comment by squidfunk** [comment, marked as answer]
Please [read our documentation](https://squidfunk.github.io/mkdocs-material/reference/annotations/#in-everything-else). Reason: [lists are not supported by the attribute lists extension](https://python-markdown.github.io/extensions/attr_list/#limitations).

**Reply by uPagge** [reply]
Eh this is very sad. sorry for my inattention, didn't notice it in the documentation

**Reply by squidfunk** [reply]
Yes, but as mentioned in the docs, it's solvable with:

``` html
<div class="annotate" markdown>

- test (1)
- test 2

</div>

1. Annotation text

```
